trajectory_tracker/src/trajectory_server.cpp

```diff
@@ -226,24 +225,27 @@ void ServerNode::updateIM()
     menu.title = "Add";
 
     mark.menu_entries.push_back(menu);
-    srv_im_fb_.insert(mark, boost::bind(&ServerNode::processFeedback, this, _1));
+    using std::placeholders::_1;
+    srv_im_fb_->insert(mark, std::bind(&ServerNode::processFeedback, this, _1));
     viz.markers.push_back(mark);
   }
-  srv_im_fb_.applyChanges();
+  srv_im_fb_->applyChanges();
 }
 
-bool ServerNode::change(trajectory_tracker_msgs::ChangePath::Request& req,
-                        trajectory_tracker_msgs::ChangePath::Response& res)
+void ServerNode::change(trajectory_tracker_msgs::srv::ChangePath::Request::SharedPtr req,
+                        trajectory_tracker_msgs::srv::ChangePath::Response::SharedPtr res)
 {
-  req_path_ = req;
-  res.success = false;
+  req_path_ = *req;
```

Comment:
preffer to use shared pointer for `req_path_`
```suggestion
  req_path_ = req;
```